src/wrapper/data_chunk.mli

```diff
@@ -1,4 +1,5 @@
 open! Core
+open! Ctypes
```

Comment:
In the wrapper library, let's avoid opening the ctypes library in the mli, so it's more explicit when there are low-level ctypes-related machinery exposed.